database.go

```diff
@@ -22,6 +23,7 @@ type Database struct {
 	ter  sql.Table
 	br   sql.Table
 	or   sql.Table
+	rer  sql.Table
```

Comment:
can we get some readable table names, please? 🙏

Reply:
I've created an issue to change all those names:

https://github.com/src-d/gitquery/issues/20